app/assets/stylesheets/widgets/home_page/blog_stories.scss

```diff
@@ -242,7 +242,7 @@
         display: inline-block;
         padding: 5px 10px 5px 20px;
         /*background:#f7f6f0;*/
-        /*border-left:5px solid #a82439;*/
+        /*border-left:5px solid #98002e;;*/
```

Comment:
@tatwell I usually don't remove any comments in case anyone put them there for a reason. Should we make it standard practice to remove dead code when we see them?